KestrelMock.Tests/MockServiceTest.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.IO;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Extensions.Configuration;
+
+namespace KestrelMock.Tests
+{
+    public class MockTestApplicationFactory
+    : WebApplicationFactory<TestStartup>
+    {
+        protected override IWebHostBuilder CreateWebHostBuilder()
+        {
+            return WebHost.CreateDefaultBuilder()
+                .UseContentRoot(Directory.GetCurrentDirectory())
+                .UseUrls("http://*:60000/")
+                .UseEnvironment("IntegrationTestEnvironment")
+                .ConfigureAppConfiguration(cfg =>
+                {
+                    cfg.AddJsonFile("appsettings.json").AddEnvironmentVariables();
+                })
+                .UseUrls("http://*:60000/")
```

Comment:
This looks like a dupe setting

Reply:
When i have time i would like to take some time to extract some service and make maybe a bit smaller classes to increase maintainability, but overall i am very happy to contribute to this project! ⭐